package.json

```diff
@@ -56,7 +55,8 @@
     "prettier": "2.0.5"
   },
   "peerDependencies": {
-    "mongodb": "3.x.x"
+    "mongodb": "3.x.x",
+    "mongodb-memory-server": "*"
```

Comment:
What was the reason of moving this out to `peerDependencies`? I'm ok with that as long as  there is a reason and this is documented in READMe.

Reply:
so we can use any version of mongodb-memory-server